plugin/src/main/groovy/com/novoda/buildproperties/BuildProperties.groovy

```diff
@@ -67,6 +70,23 @@ class BuildProperties {
         newChain(project)
     }
 
+    EntriesChain file(File file, String description = '') {
```

Comment:
Does annotating with `@Deprecated` or using the Javadoc `@deprecated` tag not mark this up in the IDE too?

Reply:
it wouldn't hurt, but unfortunately the IDE support for buildscripts is probably not gonna work fully for this. Let's add it though 👍

Reply:
Unfortunately not really since this api is exposed as DSL and will be just used in a gradle script. Otherwise it would make sense

Reply:
ok, done